doc/source/examples/ConformersMultipleMolecules/ConformersMultipleMolecules.rst

```diff
@@ -15,7 +15,7 @@ Conformers Generation with Multiple Molecules
 
 Example illustrating how to generate conformers with AMS.
 
-**This example is only compatible with AMS2024**.
```

Comment:
Not related to the PR?

Reply:
Not strictly, but was reviewing this example as it also uses the `JobAnalysis`